### PR TITLE
Readme improvements

### PR DESCRIPTION
**Pull request by sigaloid**

### CODE REVIEW

**Comment by sigaloid** [comment]
Warning: I changed `pip` to `pip3` in the installation instructions. I don't know how it works on windows but pip on my machine loads python 2.7 and this project is designed for 3. Edit as necessary.

File: README.md
Comment (nechris) [inline README.md:204]: I don't think this is accurate anymore. `requirements.txt` specifies `sciPy=1.6.1` which requires `python > 3.7`

Comment (sigaloid) [reply README.md:204]: Shall I add that to this PR?

Comment (nechris) [reply README.md:204]: Thought it makes sense as you're already update/improve the readme.

Comment (sigaloid) [reply README.md:204]: Can't update this PR anymore since I accidentally deleted the fork repo (meant to delete a fork of another repo from DidierRLopes). I'll wait for this to get merged/not merged before making another to avoid merge conflicts..

Comment (DidierRLopes) [reply README.md:204]: Hey @nechris , well spotted. This is because we don't have smoke-tests yet in place and I introduced a regression when reviewing a PR.

The code has indeed been tested with 3.6.8, but the scipy version was 1.4.1. Sorry for that! This will be fixed really soon :)

In the future we intend on not having such silly mistakes. Bear with us for the moment!

File: README.md
Comment (DidierRLopes) [inline README.md:187]: I would use: Command|Technique|Source, instead.

Love the source addition! Would also call it Wiki, Investopedia instead.

**Review by DidierRLopes — CHANGES_REQUESTED** [review]
In general I really like what you did here! Thank you for this!

I have some remarks/personal opinions that I have commented where applicable. Let me know what are your inputs if you think it's best another way.

One more idea that you gave me is adding the hyperlink from each command to their respective README and section directly. But then I "imagined" it, and it may look too much links in the same area, which doesn't look as clean!

**Comment by sigaloid** [comment]
Bad news. I deleted that fork by accident and can't make new changes. I'll attempt to move this into a new PR but until then don't close this one.

**Comment by DidierRLopes** [comment]
> Bad news. I deleted that fork by accident and can't make new changes. I'll attempt to move this into a new PR but until then don't close this one.

No problem! You should be able to copy the Raw file, so that should be easy. Just create a new PR, and we can discuss on that one how's the best way to present the README.

**Pull request closed without merging**
